pkg/policy/l3.go

```diff
@@ -22,24 +22,32 @@ import (
 
 	"github.com/cilium/cilium/api/v1/models"
 	"github.com/cilium/cilium/pkg/byteorder"
+	"github.com/cilium/cilium/pkg/labels"
 	"github.com/cilium/cilium/pkg/maps/cidrmap"
 	"github.com/cilium/cilium/pkg/policy/api"
 )
 
+// L3PolicyMapRule holds a L3 prefix and the rule labels that allow it.
+type L3PolicyMapRule struct {
+	Prefix           net.IPNet
+	DerivedFromRules []labels.LabelArray
```

Comment:
Let's declare a type for this. `LabelArrayList` or `LabeLArraySlice` or something else

Reply:
done